cvmfs/server/cvmfs_server_coda.sh

```diff
@@ -23,6 +23,10 @@ DEFAULT_LOCAL_STORAGE="/srv/cvmfs"
 
 LATEST_JSON_INFO_SCHEMA=1
 
+# Should we publish CVMFS and OS versions in meta.json?
+# Set to any non-true string to turn off this feature.
+PUBLISH_VERSIONS_IN_META_FILE=true
```

Comment:
As this variable is supposed to be overridable in the repository configuration files, we should be consistent and rename it to start with `CVMFS_`.